packages/components/src/TabBar/TabBar.scss

```diff
@@ -0,0 +1,22 @@
+.tc-tab-bar {
+	background-color: tint($brand-primary, 90);
+
+	:global(.tc-tab-bar-action) {
+		color: tint($black, 40);
+
+		&:hover,
+		&:focus {
+			background-color: tint($brand-primary, 85);
+		}
+
+		&:global(.active) {
+			background-color: tint($brand-primary, 80);
```

Comment:
guideline : `background: theme color with 0.25 opacity, ` --> `tint($brand-primary, 75)` would be closer
Don't forget to change the hover/focus below to 70